flowtorch/parameters/dense_autoregressive.py

```diff
@@ -45,6 +45,7 @@ def _build(
     ) -> None:
         # Work out flattened input and output shapes
         param_shapes_ = list(param_shapes)
+        # Why not just (sum(input_shape))?
```

Comment:
That's a good question! I can't say the following line is strictly necessary, although it may have something to do with having correct types if it is